package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "ecommerce-app",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "nodemon ./bin/www",
+    "db:makemigrations": "./node_modules/sequelize-auto-migrations/bin/makemigration.js"
+  },
+  "dependencies": {
+    "basic-auth": "^2.0.1",
+    "connect-session-sequelize": "^6.1.1",
+    "cookie-parser": "^1.4.4",
+    "cors": "^2.8.5",
+    "debug": "^2.6.9",
+    "dotenv": "^8.2.0",
+    "express": "^4.16.4",
+    "express-session": "^1.17.0",
+    "http-errors": "^1.6.3",
+    "jade": "^1.11.0",
```

Comment:
Why jade?

Reply:
came with the express boiler plate. I'll remove it